configure.ac

```diff
@@ -485,6 +485,11 @@ AS_IF([test "x$have_p11kit" = "xyes"], [
 
 AM_CONDITIONAL([HAVE_P11KIT], [test "x$have_p11kit" = "xyes"])
 
+# check for cross-compilation and set makefile flag HAVE_CROSSCOMPILE if found
+#   do not change library install location if cross-compiling
+#   and install into toolchain location provided with --prefix
+AM_CONDITIONAL([HAVE_CROSSCOMPILE], [test ! -z "$(prefix)"])
```

Comment:
`--prefix` does not mean you're cross compiling. It means you just need a different install location.

Reply:
Thanks for the feedback, with a clearer picture I have made changes to naming and the commit message